interpreter/k8s/src/main/scala/io/buoyant/transformer/k8s/DaemonSetTransformerInitializer.scala

```diff
@@ -34,7 +35,7 @@ case class DaemonSetTransformerConfig(
   override def mk(): NameTreeTransformer = {
     val client = mkClient(Params.empty).configured(Label("daemonsetTransformer"))
     def mkNs(ns: String) = Api(client.newService(dst)).withNamespace(ns)
-    val namer = new EndpointsNamer(Path.empty, mkNs)
+    val namer = new EndpointsNamer(Path.empty, labelSelector, mkNs)
```

Comment:
I think setting a labelSelector on this EndpointsNamer is going to cause problems.  The only name we ever bind with this namer is the hardcoded name on the next line `/ns/port/service` which does not include a label value.